test/sql/copy.sql.in

```diff
@@ -111,3 +111,273 @@ COPY hyper TO STDOUT DELIMITER ',';
 -- notice.
 COPY (SELECT * FROM hyper) TO STDOUT DELIMITER ',';
 
+----------------------------------------------------------------
+-- Testing multi-buffer optimization.
+----------------------------------------------------------------
+
+CREATE TABLE "hyper_copy" (
+    "time" bigint NOT NULL,
+    "value" double precision NOT NULL
+);
+
+SELECT create_hypertable('hyper_copy', 'time', chunk_time_interval => 10);
+
+-- First copy call with default client_min_messages, to get rid of the 
+-- building index "_hyper_XXX_chunk_hyper_copy_time_idx" on table "_hyper_XXX_chunk" serially
+-- messages
+\copy hyper_copy FROM data/copy_data.csv WITH csv header;
+
+SET client_min_messages TO DEBUG1;
```

Comment:
Hmm not a fan of this. This has the potential to result in flaky tests because it will also enable debug messages from postgres itself. Do we really need this or is there another way to get this information inside the test.